js/private/image/index.ts

```diff
@@ -354,22 +379,29 @@ export async function build(
                 stats.size = replaced.byteLength
             }
 
-            await add_file(key, stream, output, stats)
+            await add_file(key, stream, output, owner, stats)
         }
     }
 
     output.finalize()
 }
 
 if (import.meta.url === pathToFileURL(process.argv[1]).href) {
-    const [entriesPath, outputPath, compression, useLegacySymlinkDetection] =
-        process.argv.slice(2)
+    const [
+        entriesPath,
+        outputPath,
+        compression,
+        owner,
+        useLegacySymlinkDetection,
+    ] = process.argv.slice(2)
     const raw_entries = await readFile(entriesPath)
     const entries: Entries = JSON.parse(raw_entries.toString())
+    const [uid, gid] = owner.split(':').map(parseInt)
```

Comment:
use `Number` instead of `parseInt` so the array index doesn't get passed as the `parseInt(, base)` param.

Reply:
Done